lib/AST/Type.cpp

```diff
@@ -4137,3 +4137,14 @@ Type TypeBase::openAnyExistentialType(ArchetypeType *&opened) {
   opened = ArchetypeType::getOpened(this);
   return opened;
 }
+
+Requirement Requirement::getCanonicalRequirement() const {
```

Comment:
I suggest just putting this in the ABI diff tool. If we end up needing it again we can move it to the class.